cmd/state-installer/cmd.go

```diff
@@ -189,8 +189,8 @@ func main() {
 				Name:  "version", // note: no shorthand because install.sh uses -v for selecting version
 				Value: &params.showVersion,
 			},
+			{Name: "non-interactive", Shorthand: "n", Hidden: true, Value: &params.nonInteractive}, // don't prompt
```

Comment:
We should also auto-detect this if the user did not provide it. eg.

https://github.com/ActiveState/cli/blob/4a1a8585aa5c3b8805e26c4b274a043631b52ac2/cmd/state/output.go#L57-L57